package.json

```diff
@@ -234,6 +234,7 @@
     "release:mac": "export NODE_ENV_ELECTRON_VITE=production && vite build && electron-builder -p always --mac --x64 --arm64",
     "release:win": "cross-env NODE_ENV_ELECTRON_VITE=production vite build && electron-builder -p always --win nsis",
     "dist:linux": "export NODE_ENV_ELECTRON_VITE=production && vite build && electron-builder --linux",
+    "dist:linux:ci": "export NODE_ENV_ELECTRON_VITE=production && vite build && electron-builder --linux deb rpm pacman tar.xz --publish=never",
```

Comment:
I believe publish is never by default, no?
Since the other dist commands do not use it?

Reply:
looks like the default in github runner would be `onTagOrDraft` actually. https://www.electron.build/configuration/publish

so if we have a draft release created, our dist gh actions would publish to the draft release. 

i think we should actually change all of the dist scripts to publish never or just have one script for both release and dist and rely on the automatic rules. I'll go with the former since it is simpler